wavs.toml

```diff
@@ -21,9 +21,17 @@
 log_level = ["info", "wavs=debug"]
 
 # The IPFS gateway URL used to access IPFS content over HTTP.
-# ipfs_gateway = "https://ipfs.io/ipfs/"
+ipfs_gateway = "http://127.0.0.1:8080/ipfs/"
+# ipfs_gateway = "https://gateway.pinata.cloud/ipfs/" # testnet / production
+
+# The domain used for the warg wasi registry
+registry_domain = "localhost:8090"
+# registry_domain = "wa.dev" # testnet / production
```

Comment:
This is a major pain point for moving from local -> testnet. Do we need this to be a config value? Why not just CLI arguments only?